src/main/java/nl/cherement/jak/config/WebSocketConfig.java

```diff
@@ -0,0 +1,24 @@
+package nl.cherement.jak.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+
+    @Override
+    public void configureMessageBroker(MessageBrokerRegistry config) {
+        config.enableSimpleBroker("/");
+        config.setApplicationDestinationPrefixes("/app");
+    }
+
+    @Override
+    public void registerStompEndpoints(StompEndpointRegistry registry) {
+        registry.addEndpoint("/ws").setAllowedOrigins("*");
```

Comment:
Moet de allowedOrigins niet alleen de frontend zijn

Reply:
Zou opzich kunnen. Dan zouden we dat in de application.properties moeten zetten. En dit ook bij de andere CORS regristraties moeten aanpassen. Maar tbh maakt t niet echt uit naar mijn mening. Het ergste wat je ermee kan doen is iemand de cards op z'n board laten reloaden.